jobs/clean_builds.groovy

```diff
@@ -1,33 +0,0 @@
-import util.Common
-import org.yaml.snakeyaml.Yaml
-Common.makeFolders(this)
-
-def scipipe = new Yaml().load(readFileFromWorkspace('etc/scipipe/build_matrix.yaml'))
-
-import util.CleanBuild
-
-[
-  [
-    name: 'scipipe/lsst_distrib',
```

Comment:
This job is essential and has to stay.  These are the nightly "clean builds" that help us detect impacts from conda dependency changes.